controllers/users.js

```diff
@@ -40,4 +40,19 @@ module.exports = {
       next(httpError)
     }
   }),
+  login: catchAsync(async (req, res, next) => {
```

Comment:
Fran, aca lo mismo para mi hay que tener un auth controller, donde vamos a tener log in, register, profile. No mezclarlo con user controller.

Reply:
No, esto esta bien que este aca.